utils/utils.jsx

```diff
@@ -1857,13 +1868,58 @@ export function applyHotkeyMarkdown(e) {
     let newEnd = 0;
     if (hasItalicAndBold || (hasCurrentMarkdown && !isItalicFollowedByBold)) {
         // message already has the markdown; remove it
-        newValue = prefix.substring(0, prefix.length - delimiter.length) + selection + suffix.substring(delimiter.length);
-        newStart = selectionStart - delimiter.length;
-        newEnd = selectionEnd - delimiter.length;
+        newValue = prefix.substring(0, prefix.length - delimiterStart.length) + selection + suffix.substring(delimiterEnd.length);
+        newStart = selectionStart - delimiterStart.length;
+        newEnd = selectionEnd - delimiterStart.length;
+    } else if (isLink) {
+        if (value.length === 0) {
+            // no input; Add [|](url)
+            newValue = delimiterStart + delimiterEnd;
+            newStart = 1;
```

Comment:
Could you use `delimiterStart.length` for these instead of the plain numbers?

Reply:
Done.